infrastructure/lookerstudio/README.md

```diff
@@ -1,7 +1,16 @@
-# Marketing Analytics Looker Studio Dashboard
+# Marketing Data Engine Looker Studio Dashboard
 
+This Python script generates a URL to deploy a copy of the Marketing Data Engine dashboard utilizing the [Looker Studio Linking API](https://developers.google.com/looker-studio/integrate/linking-api).
 
+The dashboard deployment is intended to be completed once the Marketing Data Engine warehouse has been deployed through Dataform.
+
+The libraries required are in requirements.txt.
+
+You will need to configure config.ini with the BigQuery project and datasets for your warehouse.
 
 ## Run the walkthrough tutorial
 
+Click below to open the deployment tutorial in the Google Cloud console.
```

Comment:
LInk to the markdown document for the end user to follow: lookerstudio-deployment-walkthrough.md

Reply:
Done as part of refactor.